lightblue/src/test/java/org/esbtools/eventhandler/lightblue/BulkLightblueRequesterTest.java

```diff
@@ -105,6 +107,41 @@ public void shouldThrowNoSuchElementExceptionIfResponseNotFound() throws Executi
     }
 
     @Test
+    public void shouldThrowTimeoutExceptionIfResponseTimeExceedsLimit()
+            throws ExecutionException, InterruptedException, LightblueException, TimeoutException {
+
+        SlowDataLightblueClient slowClient = new SlowDataLightblueClient(client);
+        requester = new BulkLightblueRequester(slowClient);
+
+        insertUser("cooltester2000");
+
+        slowClient.pauseBeforeRequests();
+        DataFindRequest findTester = findUserByUsername("cooltester2000");
+
+        expectedException.expect(Matchers.instanceOf(TimeoutException.class));
+
+        requester.request(findTester).transformSync(responses -> {
+            return responses.forRequest(findTester).parseProcessed(TestUser.class);
+        }).get(5000, TimeUnit.MILLISECONDS);
```

Comment:
Just for the sake of quicker tests I think you can make this shorter, for example just 50 milliseconds is fine even.